src/main/scala/uk/gov/homeoffice/drt/utils/DateUtil.scala

```diff
@@ -0,0 +1,39 @@
+package uk.gov.homeoffice.drt.utils
+
+import java.text.SimpleDateFormat
+import java.time.format.DateTimeFormatter
+import java.time.{LocalDate, LocalDateTime, ZoneId, ZoneOffset}
+import java.util.Date
+
+object DateUtil {
+
+  val parseDate: String => Date = date => new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").parse(date)
+
+  val formatDate: Date => String = date => new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(date)
```

Comment:
This name should reflect the intended usage as it's not clear what the 'format' is. Is it for displaying the date in a human readable format? Would `toString` be more appropriate maybe?